src/commands.py

```diff
@@ -257,11 +257,11 @@ def secondsToHuman(seconds):
 			if not "groups" in self.wrapper.permissions: self.wrapper.permissions["groups"] = {}
 			if not "users" in self.wrapper.permissions: self.wrapper.permissions["users"] = {}
 			if not "Default" in self.wrapper.permissions["groups"]: self.wrapper.permissions["groups"]["Default"] = {"permissions": {}}
-			if player.isOp():
+			if True:
```

Comment:
Yea, that is a left over of a change I made so that non-op players could run the basic "perms" command.  I moved the player.isOp() to each of the commands that I only want "OP" to run.  Since this change is peculiar to my server only, that should probably be changed back to `if player.isOp()`  I think that is why I just made it "true" versus re-indenting everything.. So that it could be easily undone.